### PR TITLE
[VideoPlayer] enable teletext support for inputstream addons

### PR DESCRIPTION
**Pull request by phunkyfish**
## Description

Enable teletext support for inputstream addons.

## Motivation and Context

It's was missing ;)

## How Has This Been Tested?

On a new inputstream addon based on ffmpeg.

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [X] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [X] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamAddon.cpp [hunk 1]
```diff
@@ -403,11 +408,19 @@ CDemuxStream* CInputStreamAddon::GetStream(int streamId) const
     CDemuxStreamSubtitle *subtitleStream = new CDemuxStreamSubtitle();
     demuxStream = subtitleStream;
   }
+  else if (stream.m_streamType == INPUTSTREAM_INFO::TYPE_TELETEXT)
+  {
+    CDemuxStreamTeletext *teletextStream = new CDemuxStreamTeletext();
```
Comment (Rechi) [inline xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamAddon.cpp]: > Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types

**Review by Rechi — COMMENTED** [review]
Please follow the [current code guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md).

File: xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamAddon.cpp (hunk 1 above)
Comment (phunkyfish) [reply xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamAddon.cpp]: Fixed, thanks for pointing out @Rechi

**Comment by FernetMenta** [comment]
lgtm

**Comment by phunkyfish** [comment]
Cool, will merge in a couple of days if no other objections.

**Pull request merged**